### PR TITLE
8339834: Replace usages of -mx and -ms in some tests

### PR DESCRIPTION
**Pull request by jaikiran**
Can I please get a review of this trivial change which replaces the usages of `-mx` and `-ms` to `-Xmx` and `-Xms` in tests and in one code comment?

As noted in https://bugs.openjdk.org/browse/JDK-8339834, these options are outdated and support for them will soon be deprecated and removed as part of https://bugs.openjdk.org/browse/JDK-8286851.

There are some more tests remaining in client-libs area which too require a similar change. I'll be creating a separate JBS issue and PR for that.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8339834](https://bugs.openjdk.org/browse/JDK-8339834): Replace usages of -mx and -ms in some tests (**Sub-task** - P4)

### Reviewers
 * [David Holmes](https://openjdk.org/census#dholmes) (@dholmes-ora - **Reviewer**) Review applies to [8214fdb5](https://git.openjdk.org/jdk/pull/20930/files/8214fdb52eef14255b200f7a8fb419973c6070eb)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Anthony Scarpino](https://openjdk.org/census#ascarpino) (@ascarpino - **Reviewer**)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/20930/head:pull/20930` \
`$ git checkout pull/20930`

Update a local copy of the PR: \
`$ git checkout pull/20930` \
`$ git pull https://git.openjdk.org/jdk.git pull/20930/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 20930`

View PR using the GUI difftool: \
`$ git pr show -t 20930`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/20930.diff">https://git.openjdk.org/jdk/pull/20930.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/20930#issuecomment-2340196239)

### CODE REVIEW

**Review by dholmes-ora — APPROVED** [review]

File: test/jdk/java/beans/Introspector/Test5102804.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
  * @bug 5102804
  * @summary Tests memory leak
  * @author Sergey Malenkov
```
Comment (aivanov-jdk) [inline test/jdk/java/beans/Introspector/Test5102804.java:28]: Could you also remove the `@author` tag?

File: test/jdk/java/beans/Introspector/Test8027905.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
  * @bug 8027905
  * @summary Tests that GC does not affect a property type
  * @author Sergey Malenkov
```
Comment (aivanov-jdk) [inline test/jdk/java/beans/Introspector/Test8027905.java:30]: Could you also remove the `@author` tag?

**Review by aivanov-jdk — APPROVED** [review]
Looks good to me.

It could be good to remove `@author` tags from all the modified tests.

File: test/jdk/java/beans/Introspector/Test5102804.java (hunk 1 above)
Comment (jaikiran) [reply test/jdk/java/beans/Introspector/Test5102804.java]: Done - I've now updated the PR to remove these author tags. Tests continue to pass.

**Review by aivanov-jdk — APPROVED** [review]

**Review by ascarpino — APPROVED** [review]
The Cache.java change, and the others, look good to me.

**Review by prrace — APPROVED** [review]

**Comment by jaikiran** [comment]
Thank you everyone for the reviews.

**Pull request closed without merging**